flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-table</artifactId>
+		<version>1.8-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>flink-table-planner-blink_${scala.binary.version}</artifactId>
+	<name>flink-table-planner-blink</name>
+	<description>
+		This module bridges Table/SQL API and runtime. It contains
+		all resources that are required during pre-flight and runtime
+		phase. The content of this module is work-in-progress. It will
+		replace flink-table-planner once it is stable. See FLINK-11439
+		and FLIP-32 for more details.
+	</description>
+
+	<packaging>jar</packaging>
+
+	<dependencyManagement>
+		<dependencies>
+			<!-- Common dependency of calcite-core and flink-* -->
+			<dependency>
+				<groupId>com.google.guava</groupId>
+				<artifactId>guava</artifactId>
+				<version>19.0</version>
```

Comment:
Does it make sense to use property like guava.version, to keep all version synced.

Reply:
Oh, i just noticed guava version has been defined in root pom, and the version is 18.0. Do we need 19.0 for table?

Reply:
Calcite does only support a certain range of Guava versions. I think it makes sense to hard code the version here next to Calcite. About Janino we just had a discussion in #7407 but I like the idea of defining the Janino version globally in a variable. Will do that.